### ISSUE TITLE
Ledger only showing one account

### ISSUE THREAD

**Issue by Mshuu** [Mshuu/SwitchDexIssues#10]

**Comment by snowkidind** [comment]
Yes, cannot select account when connected to ledger nano x as well. So all my funds are not on the default address, and Mdex uses the first address that basically shows up in MEW, where ledger live doesn't use that as default anymore, so most definitely need to be able to select an appropriate address when connecting with ledger.
